app/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/BaseConnector.scala

```diff
@@ -28,26 +28,26 @@ trait BaseConnector {
   def appConfig: AppConfig
   def dateTimeService: DateTimeService
 
-  def buildHeaders(correlationId: String, accessToken: String)(implicit hc: HeaderCarrier): Seq[(String, String)] =
+  protected def buildHeaders(correlationId: String, accessToken: String, forwardedHost: String)(implicit
+    hc: HeaderCarrier
+  ): Seq[(String, String)] =
     Seq(
       HeaderNames.CorrelationId -> correlationId,
-      HeaderNames.ForwardedHost -> appConfig.eisConfig.forwardedHost,
+      HeaderNames.ForwardedHost -> forwardedHost,
```

Comment:
think forwarded host is same for both, may be pass directly from config (hawk and pega) if its different then we can remove appConfig from  declaration at line 28

Reply:
Whilst I can't see B&T and Case Management needing separate ones in the future, I'm happy for this to remain as two distinct configs for now